libs/execution_context/ebpf_link.c

```diff
@@ -154,10 +164,23 @@ _ebpf_link_extension_changed_callback(
 
     // Complete detach.
     if (provider_data == NULL) {
+        // Check if the link is in the process of attaching.
+        if (link->state == EBPF_LINK_STATE_DETACHING) {
```

Comment:
The comment above this line mentions 'attaching' but we're checking for _detaching_? (`EBPF_LINK_STATE_DETACHING`

Reply:
Agreed. Comment is wrong.